app/assets/stylesheets/base.scss

```diff
@@ -0,0 +1,61 @@
+a {
+  &.disabled {
+    color: $gray-ccc;
+    cursor: not-allowed;
+    text-decoration: none;
+  }
+}
+
+body {
+  font-family: 'Source Sans Pro', 'Arial Unicode MS', Helvetica, sans-serif;
+}
+
+a, a:hover, a:focus {
+  text-decoration: none;
```

Comment:
Properties should be ordered border-bottom, text-decoration<br>Each selector in a comma sequence should be on its own line